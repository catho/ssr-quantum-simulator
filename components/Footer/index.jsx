import Container from '@catho/quantum/Grid/sub-components/Container';

const Footer = () => (
  <Container className="Footer-text">
    <p>
      Essa tela foi especialmente desenvolvida para a dev.Ela maravilhosa da
      ACME.
    </p>
    <p>
      <time dateTime="2023-02-09">9 de fevereiro de 2023</time>
    </p>
    <p>Por designer com m√£o de android</p>
  </Container>
);

export default Footer;
