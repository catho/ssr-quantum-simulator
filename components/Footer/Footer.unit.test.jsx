import { render } from '@testing-library/react';
import Footer from '.';

test('should match the snapshot', () => {
  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
