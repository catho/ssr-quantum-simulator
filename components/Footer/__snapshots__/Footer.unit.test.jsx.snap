// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div>
  <div
    class="Container-sc-ka34zl-0 gDxJPl Footer-text"
  >
    <p>
      Essa tela foi especialmente desenvolvida para a dev.Ela maravilhosa da ACME.
    </p>
    <p>
      <time
        datetime="2023-02-09"
      >
        9 de fevereiro de 2023
      </time>
    </p>
    <p>
      Por designer com m√£o de android
    </p>
  </div>
</div>
`;
